### PR TITLE
Add current-bench for deployments

### PR DESCRIPTION
**Pull request by gs0510**
Current bench is currently deployed on the autumn machine which
has been configured to run the benchmarks, so also adding autumn
to the list of machines.

### CODE REVIEW

File: src/pipeline.ml [hunk 1]
```diff
@@ -90,6 +90,7 @@ module Cluster = struct
   module Ci4_docker = Current_docker.Make(struct let docker_context = Some "ci4" end)
   module Ci6_docker = Current_docker.Make(struct let docker_context = Some "docsci" end)
   module Toxis_docker = Current_docker.Make(struct let docker_context = Some "toxis" end)
+  module Cb_docker = Current_docker.Make(struct let docker_context = Some "cb" end)
```
Comment (talex5) [inline src/pipeline.ml:93]:
```suggestion
  module Autumn_docker = Current_docker.Make(struct let docker_context = Some "autumn-current-bench" end)
```
cb seems to be a different machine:
```
cb.ocamllabs.io has address 51.158.77.72
autumn.ocamllabs.io has address 128.232.124.187
```

File: src/pipeline.ml [hunk 2]
```diff
@@ -222,6 +224,11 @@ let v ~app ~notify:channel ~sched ~staging_auth () =
         docker "docker/storage/Dockerfile"  ["live", "ocurrent/docs-ci-storage-server:live", [`Ci6, "infra_storage-server"]];
         docker "Dockerfile.web"             ["live-web", "ocurrent/docs-ci-web:live", [`Ci6, "infra_docs-ci-web"]];
       ];
+      ocurrent, "current-bench", [
+        docker "pipeline/Dockerfile" ["live", "ocurrent/current-bench-pipeline", [`Cb, "current-bench-pipeline"]];
```
Comment (talex5) [inline src/pipeline.ml:228]: Are you sure these service names are correct? There is normally a `_` somewhere, separating the stack name from the service name.

**Review by talex5 — COMMENTED** [review]
You'll also need to add this to `.ssh/authorized_keys` so that the deployer can connect:
```
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAyRYpH3cy4s0t/yXM27c5quXwTC4+vaqGl43jaA9EMxnHLlQLP69aixi9/6Y0jN3RgNsL8CdOHKXNdKet2o/uqK7p8YQnrXKgoYooCNS+vg0NAL0QqE/UMryBv0nFa6PIKeWmjId4qoKd/tlkkvBdtuhBk4cg7K8wQYzqC4eJC8Iby4ZswadZl2GW2LiGuj3CAS6XuAD9dhVpZJrOcl+5RyHMnxT8J4iM8JCI2hlmp1+D1UGmaDjuPc7IChxx9+zx0XsYHqAkNoxFGYoI6IejcTIF9VGKdHtUOFcRIi3F50zRTKqLLyE7Wx9XiID13NVIyJOsLQB/MPGbcOCPWmsp
```

**Comment by gs0510** [comment]
Thanks for the review @talex5! Fixed the names and added the authorized key for the deployer :)

**Comment by talex5** [comment]
I've created these for the images:
- https://hub.docker.com/repository/docker/ocurrent/current-bench-frontend
- https://hub.docker.com/repository/docker/ocurrent/current-bench-pipeline

Once the service stack is running (running `docker stack ls` should show it) I'll deploy this to keep the stack up-to-date.

**Pull request merged**